src/james.format.xml.clss.pas

```diff
@@ -39,20 +39,24 @@ interface
 
 type
   {$IFDEF FPC}
-  TXMLDocument = Laz2_DOM.TXMLDocument;
-  TDOMNode = Laz2_DOM.TDOMNode;
+    TXMLDocument = Laz2_DOM.TXMLDocument;
+    TDOMNode = Laz2_DOM.TDOMNode;
   {$ENDIF}
 
   TXMLComponent = class
```

Comment:
I agree. Unfortunately XML handling has some substantial differences between Delphi and FPC. 
I have refactored TXMLComponent lightly to accommodate some of those differences, but a deeper refactoring is needed to clean up the code.